src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -574,6 +588,28 @@ JSC::EncodedJSValue SOCKET_BAD_PORT(JSC::ThrowScope& throwScope, JSC::JSGlobalOb
     return {};
 }
 
+JSC::EncodedJSValue UNCAUGHT_EXCEPTION_CAPTURE_ALREADY_SET(JSC::ThrowScope& throwScope, JSC::JSGlobalObject* globalObject)
+{
+    auto message = makeString("`process.setupUncaughtExceptionCapture()` was called while a capture callback was already active"_s);
```

Comment:
`makeString` allocates a String. We don't need to allocate the string here - just the string literal is fine